test/TravelersRepository-test.js

```diff
@@ -38,6 +38,14 @@ describe('Travelers Repository', () => {
     expect(travelersRepository.allTravelers).to.be.an('array');
   });
 
+  it('should hold all Trips', () => {
```

Comment:
could use `deep.equal` to check that the contents match what you're putting in too